### PR TITLE
Fixes for VPC-native cluster.

### PR DESCRIPTION
**Pull request by RayRoestenburg**
The GKE Ingress only functions with ClusterIP if ip-aliasing is enabled.

### CODE REVIEW

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc [hunk 1]
```diff
@@ -24,7 +24,7 @@ Once you've purchased a plan to use the Akka Cloud Platform, proceed to configur
 
 To completely configure and use the Akka Cloud Platform you need to fill in a few fields:
 
-. The GKE cluster — This is the cluster where you are installing the operator. Use "Create a new cluster" if you don't have one.
+. The GKE cluster — This is the cluster where you are installing the operator. The Akka operator requires Alias IP ranges for setting up ingress, so please setup a VPC-native cluster. Please find instructions at https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips[https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips {tab-icon}, window="tab"] to setup a VPC-native cluster. You can read about the benefits of a VPC-native cluster at https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips[https://cloud.google.com/kubernetes-engine/docs/concepts/alias-ips {tab-icon}, window="tab].
```
Comment (pyoio) [inline docs-source/docs/modules/deployment/pages/gcp-install.adoc:27]: Should we call this out as "If you want to use Akka Operator to manage ingress into your HTTP and GRPC endpoints via the cloud-native controllers"?

Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc:27]: yeah good idea

Comment (RayRoestenburg) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc]: Fixed.

**Review by andreaTP — APPROVED** [review]

File: docs-source/docs/modules/deployment/pages/gcp-install.adoc (hunk 1 above)
Comment (thothmoses) [reply docs-source/docs/modules/deployment/pages/gcp-install.adoc]: LGTM

**Review by chbatey — COMMENTED** [review]
Looks good!

**Pull request merged**